ui/app/components/mount-backend-form.ts

```diff
@@ -65,7 +67,7 @@ export default class MountBackendForm extends Component<Args> {
     const mountTypes =
       this.args.mountType === 'secret'
         ? allEngines().map((engine) => engine.type)
-        : methods().map((auth) => auth.type);
+        : mountableAuthMethods(this.version.isEnterprise).map((auth) => auth.type);
```

Comment:
previously `methods()` didn't account for enterprise methods, now it does (though I don't think  it was a big deal beforehand)